### PR TITLE
developers sso into AWS

### PR DESCRIPTION
**Pull request by tiruworktito**
## Changes
1. Changed the provisioning infrastructure markdown to include sso 

## Purpose
Documentation between the onboarding assignment and the instructions on this repo are not the same.
 
## Approach
_How does this change address the problem?_
This PR addresses the difference between the two by adding in the same instructions from the onborarding assignment. 

## Pre-Testing TODOs
you should have the laptop script installed.

## Testing Steps
run the commands introduced. 

Closes #001

### CODE REVIEW

**Review by fluturecode — DISMISSED** [review]
This looks fine to me. Nice work.

File: docs/docs/deploying-your-app/01-provisioning-infrastructure.md [hunk 1]
```diff
@@ -11,27 +11,36 @@ infrastructure. Please note that if you used our
...
+- [AWS Cli](https://aws.amazon.com/cli/) for storing your AWS
+  credentials for terraform to use.
 
 ## Quick Start
 
 In order to start provisioning your infrastructure we need to initialize
 terraform. Before getting started, make sure you have AWS cli installed
 and the `shift3` profile configured.
 
+nm-01-ssoconfigure
```
Comment (michaelachrisco) [inline docs/docs/deploying-your-app/01-provisioning-infrastructure.md]: What is this? A branch name?

File: docs/docs/deploying-your-app/01-provisioning-infrastructure.md [hunk 2]
```diff
@@ -11,27 +11,36 @@ infrastructure. Please note that if you used our
...
+```bash
+$ aws configure sso--profile shift3
 ```bash	
 $ aws configure sso --profile shift3
+main
```
Comment (michaelachrisco) [inline docs/docs/deploying-your-app/01-provisioning-infrastructure.md]: Please remove branch names from the instructions. I dont think this provides value to our developers going through the instructions.

File: docs/docs/deploying-your-app/01-provisioning-infrastructure.md [hunk 3]
```diff
@@ -11,27 +11,36 @@ infrastructure. Please note that if you used our
...
 
 SSO session name (Recommended): shift3
 SSO start URL [None]: https://bitwiseindustries.awsapps.com/start
 SSO region [None]: us-west-2
+nm-01-ssoconfigure
+SSO registration scopes [sso:account:access]:
+(press inter)
```
Comment (michaelachrisco) [inline docs/docs/deploying-your-app/01-provisioning-infrastructure.md:35]:
```suggestion
(press enter)
```

File: docs/docs/deploying-your-app/01-provisioning-infrastructure.md [hunk 4]
```diff
@@ -11,27 +11,36 @@ infrastructure. Please note that if you used our
...
+```bash
+$ aws configure sso--profile shift3
 ```bash	
 $ aws configure sso --profile shift3
+main
 
 SSO session name (Recommended): shift3
 SSO start URL [None]: https://bitwiseindustries.awsapps.com/start
 SSO region [None]: us-west-2
+nm-01-ssoconfigure
```
Comment (michaelachrisco) [inline docs/docs/deploying-your-app/01-provisioning-infrastructure.md]: Please remove branch names from the instructions.

File: docs/docs/deploying-your-app/01-provisioning-infrastructure.md [hunk 5]
```diff
@@ -11,27 +11,36 @@ infrastructure. Please note that if you used our
...
+SSO registration scopes [sso:account:access]:
+(press inter)
+To test this out
 SSO registration scopes [sso:account:access]: (press Enter)
 ```
 To test this out: 
 
+main
```
Comment (michaelachrisco) [inline docs/docs/deploying-your-app/01-provisioning-infrastructure.md]: Same here.

**Review by michaelachrisco — CHANGES_REQUESTED** [review]
Thanks for contributing @tiruworktito! Thanks for taking a look at anopther place our docs need to be updated. 

A couple of blockers: 

1. Please change the base to the `develop` branch. When we have new changes within the boilerplates, we should follow the S&P [git flow](https://github.com/Shift3/standards-and-practices#git) standards. Please do the following: 

Change_branch_base


2. Please remove the branch names from the markdown file. For example `nm-01-ssoconfigure` and `main` should be removed. 

3. Lets keep the same standard we have going in the Devops Apprentices Notes repository and include myself and the DevOps team on any new Pull Requests to the boilerplates. Please add @michaelachrisco and @BWTCDevops as reviewers. 
Let me know when you are finished with the above changes via slack or need any help. 

4. "Closes https://github.com/Shift3/dj-starter/pull/1" Please reference the issue this PR is addressing. Please change this line in your summary to "Closes #59"

Appreciate you taking the initiative updating our docs! I know the first Pull Requests can be a bit difficult but you did great jumping in.

**Review by michaelachrisco — CHANGES_REQUESTED** [review]
Thanks for dealing with the merge conflict and changing the base branch @tiruworktito !

Please take a look at the comments above for feedback. Let me know if you need any help with that or need direction!

File: docs/docs/deploying-your-app/01-provisioning-infrastructure.md (hunk 1 above)
Comment (tiruworktito) [reply docs/docs/deploying-your-app/01-provisioning-infrastructure.md]: not the name of the branch

**Pull request closed without merging**